containers/zammad/docker-entrypoint.sh

```diff
@@ -91,7 +92,7 @@ if [ "$1" = 'zammad-init' ]; then
     sleep 5
   done
 
-  if ! curl -s ${ELASTICSEARCH_HOST}:${ELASTICSEARCH_PORT}/_cat/indices | grep -q zammad; then
+  if ! curl -s -k ${ELASTICSEARCH_PROTO}://${ELASTICSEARCH_HOST}:${ELASTICSEARCH_PORT}/_cat/indices | grep -q zammad; then
```

Comment:
i don't like to have "-k" as default.
we should add another variable "SSL_VERIFY" to enable / disable "-k" via if statement.

Reply:
You're right! Should I just use `SSL_VERIFY` or `ELASTICSEARCH_SSL_VERIFY`?

Reply:
i think ELASTICSEARCH_SSL_VERIFY would fit best.

Reply:
I reworked the naming slightly to make it easier to digest – please take a look. Your example had it backwards which tripped me up for a second there: `-k` skips verification which would've been set in your example when `ELASTICSEARCH_SSL_VERIFY` is `true`. 🙈